### ISSUE TITLE
Usability: Target section selection

### ISSUE THREAD

**Issue by mudrd8mz** [corvus-albus/moodle-local_importtosection#1]
On the "Select a target section" page, ideally the whole table row or at least the topic na,me text should work as a label. So that clicking anywhere in the row (or at least the topic name) would select the particular radio button. As it is now, only the radio buttons themselves are clickable.

![image](https://user-images.githubusercontent.com/85831/87542485-4f434900-c6a3-11ea-9c26-fdae14d87e2f.png)

**Comment by corvus-albus** [comment]
Thank you for your suggestion. That would really be useful. As I'm no trainend php-developer, my main type of coding is "analog-engineering". 
This dialog is based on the ui of the well-known import-ui of moodle. And there it is just the same. 
I will have a look at it but can't promise anything.
Cheers corvus-Albus

**Comment by mudrd8mz** [comment]
> And there it is just the same.

Yeah, I realized that later during the testing, too. No worries, it's not anything crucial. Just something that caught my attention in otherwise well written plugin.

**Comment by corvus-albus** [comment]
O.k. \<label\> ... \</label\> does the trick. I saw an example with css, that allowed the whole row to be clickable, but that's out of reach for me. But with labeling I manage at least the text to be clickable - and now I see: It's included in your post already. Didn't realize "label" is a keyword. 
I will put it in soon. 
is there a possibility to change it in the import-ui as well? (That's what pull requests are for, I assume?)
Cheers corvus-albus

**Comment by mudrd8mz** [comment]
Yes, thanks. And sorry if I was not clear enough.

> is there a possibility to change it in the import-ui 

Please feel encouraged - https://docs.moodle.org/dev/Process#Fixing_a_bug (note we do not use Github pull requests for the Moodle core).

**Comment by corvus-albus** [comment]
@mudrd8mz 
As I mentioned above labeling works. But there is an unwanted side effect, which seems to belong to moodles html table definitons. 
When labeling the text the table row gets some extra space at the bottom:
Table with labels:
![with_label](https://user-images.githubusercontent.com/26668681/89021882-82cbd780-d321-11ea-9ff7-0bb858536db5.png)
Table without labels:
![without_label](https://user-images.githubusercontent.com/26668681/89021894-85c6c800-d321-11ea-8c38-0577e3760673.png)

As I try this with a "naked" html-table there is no difference in row height by labeling. I tried it with the html-code at the bottom. In addition a div in a cell in this table leads to the cell being a clickable region as a whole. In moodles table a div has no effect.

The best outcome would be, that there is no extra space through labeling and a div would make the whole cell clickable. Do you have any ideas on that?

Cheers corvus albus

(Sorry for taking you as an html-styles expert but I don't know a better place to ask. If you know one please tell me.)

```<!DOCTYPE html>

<html>
<body>
<table border="1">
    <tbody>
        <tr>
            <td class="cell c0" style=""><input type="radio" name="targetsection" value="0" id="0" /></td>
            <td class="cell c1 lastcol" style=""><label for="0"><div>General</div></label></td>
        </tr>
        <tr>
            <td class="cell c0" style=""><input type="radio" name="targetsection" value="1" id="1" /></td>
            <td class="cell c1 lastcol" style="">KlasseKlasseKlasseKlasseKlasse</td>
        </tr>
    </tbody>
</table>

</body>
</html>
